app.js

```diff
@@ -20,3 +20,265 @@ navLink.forEach((link) => link.addEventListener('click', () => {
+  const layoutClass = ['card-work', 'card-work1'];
+  layout.classList.add(layoutClass[0]);
+  layout.classList.add(layoutClass[1]);
+  layout.innerHTML = `<figure><img scr= "${work.featureImage}" class="${work.classMobileImg}">
+    <img src= "${work.featureImageDesk}" class="${work.classDeskImg}">
+    <figcaption><h2>${work.caption}</h2></figcaption></figure>
+    <div class='aside'>
+      <div class='type'>
+       <h2>${work.projectName}</h2>
+       <ul>
+          <li class="title">${work.feature.title}</li>
+          <li class="domain">${work.feature.domain}</li>
+          <li class="year">${work.feature.year}</li>
+       </ul>
+      </div>
+      <p class="description">${work.projectDescription}</p>
+      <div class="languages languages-desk">
+        <ul>
+          <li><span class="html">${work.technologies[0]}</span></li>
+            <li><span class="css">${work.technologies[1]}</span></li>
+            <li><span class="js">${work.technologies[2]}</span></li>
+          </ul>
+      </div>
+      <button class="seeProject">See project <img src= "${work.featureImageDesk}" class="${work.classDeskImg}"></button>
+    </div>`;
+  workSectionContainer.appendChild(layout);
+
+  /* Generation of the popUp container */
+  /* Generate popUp window */
+
+  const popLayer = document.querySelector('.layer');
+  const popUpWindow = document.querySelector('.popup-window');
+  const sample = document.querySelectorAll(`.${work.classDeskImg}`);
+  sample.forEach((image) => {
+    image.addEventListener('click', () => {
+      work.featureImageDesk = image.getAttribute('src');
+      popUpWindow.innerHTML = `<h2>${work.projectName}</h2>
+      <div class="header">
+        <ul class="feature">
+          <li class="title">${work.feature.title}</li>
+          <li class="domain">${work.feature.domain}</li>
+          <li class="year">${work.feature.year}</li>
+        </ul>
+        <button class="close">&times;</button>
+      </div>
+      <img src="${image.getAttribute('src')}" class="card-work" alt="" />
+      <div class="popup-body">
+        <p class="description">${popUpData[0].projectDescription}</p>
+        <div class="left">
+          <div class="popup-languages">
+            <ul>
+              <li>${work.technologies[0]}</li>
+              <li>${work.technologies[1]}</li>
+              <li>${work.technologies[2]}</li>
+            </ul>
+            <ul class="supplement-desk">
+              <li>${work.technologies[3]}</li>
+              <li>${work.technologies[4]}</li>
+              <li>${work.technologies[5]}</li>  
+            <ul/>
+          </div>
+          <div class="popup-btn">
+            <button class="live">See live <img src="./desktop-img/liveicon.png" alt="" />
+            </button>
+            <button class="source">See Source <img src="./desktop-img/sourcevector.png" alt="" />
+            </button>
+          </div>
+        </div>
+      </div>`;
+      popLayer.appendChild(popUpWindow);
```

Comment:
#### Kindly see this video below to see how your popup modal looks on the mobile and desktop version
- [ ] The buttons and li tags are not styled properly for the mobile screen size

- [ ] For desktop, at 768px, kindly increase the height  of the modal so users can see all information at one glance and the modal should be centered having same margin at the top, bottom and left, right 🤝 

https://user-images.githubusercontent.com/69638013/162404689-dd90cf0a-73a2-4ddb-82e7-996eaa376078.mp4

Reply:
responsiveness solved.
As I have mention in the first comment, if I try to put popUp right containing all the screen, the scroll will be very long that's why I try to adjust it's height for better user experience.